src/main.js

```diff
@@ -100,11 +100,19 @@ app.on('ready', async () => {
     mainWindow.setMenu(null);
 
     chain.on("start", () => {
-      chain.startServer(Settings.getAll().server)
+      let settings = Settings.getAll()
+      chain.startServer(settings.server)
     })
 
     chain.on("server-started", (data) => {
-      mainWindow.webContents.send(SET_KEY_DATA, data)
+      mainWindow.webContents.send(SET_KEY_DATA, { 
+        privateKeys: data.privateKeys,
+        mnemonic: data.mnemonic,
+        hdPath: data.hdPath
+      })
+
+      Settings._onNewMnemonic(data.mnemonic)
```

Comment:
It feels to me like the logic of `onNewMnemonic` should be pulled out to here. i.e., something like

```
if (!Settings.hasMnemonic()) {
  Settings.setMnemonic(data.mnemonic);
}
```

Reply:
Thinking about this further, I think I'm generally not in favor of pulling it from ganache-core as it's very indirect (though I'll go with it either way). The code required to create a mnemonic is pretty minimal, and the process would be more direct if Ganache created the mnemonic explictly. To reduce maintenance, we can abstract out the mnemonic creation code from ganache-core into its own library and use that instead so there's no copy-pasta.

Reply:
I gave this thought before implementing, and reached a different conclusion. If all we were pulling was the mnemonic, I'd agree. However the strategy I'm working toward, and hence the "not strictly necessary changes" above, is to make it so that it's only necessary define settings in Ganache which should differ from the `ganache-core` defaults. Since a facility to get a fleshed-out settings object needs to exist for that purpose, I felt like leveraging that for this purpose was fine.

That said, I'm open to the idea that this is the wrong facility to use. I can imagine exposing something like `evm_getOptions` might be a more desirable approach, but the one I've implemented here expanded upon an existing mechanism and required fewer releases to ship, so it felt like a natural choice.